core/src/main/java/com/digitalpebble/storm/crawler/filtering/regex/RegexURLFilterBase.java

```diff
@@ -49,22 +50,38 @@
 
     @Override
     public void configure(Map stormConf, JsonNode paramNode) {
-
-        JsonNode filenameNode = paramNode.get("regexFilterFile");
-        String rulesFileName;
-        if (filenameNode != null) {
-            rulesFileName = filenameNode.textValue();
+        JsonNode node = paramNode.get("urlFilters");
+        if (node != null && node.isArray()) {
+            rules = new ArrayList<RegexRule>();
+            ArrayNode array = (ArrayNode) node;
+            for (JsonNode urlFilterNode : array) {
+                try {
+                    RegexRule rule = createRule(urlFilterNode.asText());
+                    if (rule != null) {
+                        rules.add(rule);
+                    }
+                } catch (IOException e) {
+                    LOG.error("There was an error reading regex filter {}",
+                            urlFilterNode.asText());
+                    e.printStackTrace();
+                }
```

Comment:
I think to keep the configure method lean, it would be better to have a readRules method which takes an ArrayNode